### PR TITLE
Created placeholder cha training page 

### PR DESCRIPTION
**Pull request by viviancan**
Refs #4516 Created placeholder cha training page

### CODE REVIEW

File: app/controllers/chapter_ambassador/training_completions_controller.rb [hunk 1]
```diff
@@ -0,0 +1,5 @@
+module ChapterAmbassador
+  class TrainingCompletionsController < ChapterAmbassadorController
```
Comment (viviancan) [inline app/controllers/chapter_ambassador/training_completions_controller.rb:2]: I went with `TrainingCompletions`based on the other training controllers/naming for judges and mentors. Open to suggestions for different naming!

Comment (shaun-technovation) [reply app/controllers/chapter_ambassador/training_completions_controller.rb:2]: Cool, sounds good to me, good call on keeping it consistent!

File: app/helpers/application_helper.rb [hunk 2]
```diff
@@ -23,6 +23,7 @@ def determine_manifest
       dashboards
       location_details
       profiles
+      training_completions
```
Comment (shaun-technovation) [inline app/helpers/application_helper.rb:26]: 👍

**Review by shaun-technovation — APPROVED** [review]
Cool, looks good/straight forward to me! 🙌 👏 🎉 🚢

**Pull request merged**